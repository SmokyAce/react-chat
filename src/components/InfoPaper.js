import React from 'react';
import PropTypes from 'prop-types';
import { withStyles } from 'material-ui/styles';
import Paper from 'material-ui/Paper';
import Typography from 'material-ui/Typography';

const styles = theme => ({
  paper: {
    padding: theme.spacing.unit * 3,
  },
});

const InfoPaper = ({ classes }) => (
  <Paper className={classes.paper}>
    <Typography variant="display1" gutterBottom>
      Start messagingâ€¦
    </Typography>
    <Typography variant="body1" gutterBottom>
      Use <strong>Global</strong> to explore communities around here.
    </Typography>
    <Typography variant="body1" gutterBottom>
      Use <strong>Recents</strong> to see your recent conversations.
    </Typography>
  </Paper>
);

InfoPaper.propTypes = {
  classes: PropTypes.objectOf(PropTypes.string).isRequired,
};

export default withStyles(styles)(InfoPaper);
