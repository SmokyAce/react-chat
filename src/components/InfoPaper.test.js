/* eslint-env jest */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import InfoPaper from './InfoPaper';

describe('<InfoPaper />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<InfoPaper />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders correctly', () => {
    const tree = renderer.create(<InfoPaper />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
