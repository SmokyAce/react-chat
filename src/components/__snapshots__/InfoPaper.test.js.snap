// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoPaper /> renders correctly 1`] = `
<div
  className="MuiPaper-root-54 MuiPaper-elevation2-58 MuiPaper-rounded-55 InfoPaper-paper-53"
>
  <h1
    className="MuiTypography-root-81 MuiTypography-display1-85 MuiTypography-gutterBottom-98"
  >
    Start messagingâ€¦
  </h1>
  <p
    className="MuiTypography-root-81 MuiTypography-body1-90 MuiTypography-gutterBottom-98"
  >
    Use 
    <strong>
      Global
    </strong>
     to explore communities around here.
  </p>
  <p
    className="MuiTypography-root-81 MuiTypography-body1-90 MuiTypography-gutterBottom-98"
  >
    Use 
    <strong>
      Recents
    </strong>
     to see your recent conversations.
  </p>
</div>
`;
